### ISSUE TITLE
First request returns 404 with JWT 

### ISSUE THREAD

**Issue by ppascualv** [mongodb-js/charts-embed-sdk#33]
**Describe the bug**
When charts are rendered using a JWT seem that the first fetch is returning 404 and the auth header is not present, however, it automatically does a retry with the correct header.

**To Reproduce**
I'm using React
```
  useEffect(() => {
    const newSdk = new ChartsEmbedSDK({
      baseUrl: "https://charts.mongodb.com/charts-swipcar-iukiy",
      showAttribution: false,
      theme: "light",
      getUserToken
    })
    setSdk(newSdk)
  }, [])

async function getUserToken() {
  const x = await getChartsAuthToken()
    .then(response => {
      return response.data.authToken
    })
    .catch(() => console.error("Chart auth failed"))

  return x
}
```

**Expected behavior**
They should not generate 404 requests

**Screenshots**
![image](https://user-images.githubusercontent.com/2453682/121663016-41aa6500-caa6-11eb-8a5a-4247ca0cb5c7.png)
failing request:
![image](https://user-images.githubusercontent.com/2453682/121663187-71596d00-caa6-11eb-8f27-b5613a5ab277.png)
response:
{"errorCode":3,"simple":"Error loading data for this chart (error code: 3).","verbose":"Error loading data for this chart (error code: 3). Embedding not enabled on this chart. See https://dochub.mongodb.org/core/charts-embedding-error-codes for details."}
good request:
![image](https://user-images.githubusercontent.com/2453682/121663418-b2518180-caa6-11eb-9d9e-8addcd9d3521.png)

**Desktop (please complete the following information):**
 - OS: ubuntu 21
 - Browser chrome

**Additional context**

**Comment by tomhollander** [comment]
Thanks @ppascualv - this should be fixed in the `2.1.0-beta.1` version - could you please check that and let me know if it's still an issue? We'll include this fix in the next release version.

**Comment by ppascualv** [comment]
Seems fixed @tomhollander. Thanks!
